packages/wrangler/src/__tests__/publish.test.ts

```diff
@@ -465,9 +464,8 @@ export default{
 
     it('should error if a site definition doesn\'t have a "bucket" field', async () => {
       writeWranglerToml({
-        site: {
-          "entry-point": "./index.js",
-        } as Config["site"],
+        // @ts-expect-error we're intentionally setting an invalid config
+        site: {},
```

Comment:
Did you intentionally use a TS expect error directive here rather than the cast that avoids the error?

```suggestion
        site: {} as Config["site"],
```

Reply:
Yup. What's the norm here when it's intentionally invalid?

Reply:
In tests I have preferred to just use casts as I have a phobia of `@ts...` directives :-D
But I can equally accept that it is good to be able to a) get an error if this stops being invalid, and b) have a comment explaining why this is here.
So OK, let's go with this.